src/test/java/uk/gov/companieshouse/service/dissolution/DissolutionGetterTest.java

```diff
@@ -39,9 +39,10 @@
     public static final String APPLICATION_REFERENCE = "XYZ456";
     public static final String OFFICER_ID_ONE = "abc123";
     public static final String OFFICER_ID_TWO = "def456";
+    private static final String EMAIL = "applicant@email.com";
```

Comment:
does this file need reverted?

Reply:
Since I touched the file already, sonar will complain about "public" modifiers for test classes. So I only lefte these changes and removed the others.